clientv3/watch_test.go

```diff
@@ -15,9 +15,12 @@
 package clientv3
 
 import (
+	"context"
 	"testing"
 
-	"go.etcd.io/etcd/mvcc/mvccpb"
+	"google.golang.org/grpc/metadata"
+
+	"go.etcd.io/etcd/api/v3/mvccpb"
```

Comment:
```suggestion
	"google.golang.org/grpc/metadata"

	"go.etcd.io/etcd/mvcc/mvccpb"
```

Reply:
Great catch! updated the relevant commit.